wp-content/themes/sceraTheatre/scss/blocks/_giftshop.scss

```diff
@@ -0,0 +1,19 @@
+.gift-cards{
+    padding: 6rem 1.6rem 4.8rem;
+    text-align: center;
+
+	h2, p{
+        color: $color-blue;
```

Comment:
Remove this line. h2 already has .blue on it, and you're making the <p> blue on line 12 below.